src/watcher.zig

```diff
@@ -436,6 +440,10 @@ pub fn NewWatcher(comptime ContextType: type) type {
 
         // This must only be called from the watcher thread
         pub fn watchLoop(this: *Watcher) !void {
+            if (Environment.isWindows) {
+                @panic("watchLoop should not be used on Windows");
```

Comment:
can this be a compiler error instead?

Reply:
fixed in https://github.com/oven-sh/bun/pull/8379/commits/b323bc66cfc1bb68130489c1f03e90679f6667a2